### ISSUE TITLE
500 error loading bounty list

### ISSUE THREAD

**Issue by mcronce** [bountysource/core#1076]
Happens when the AJAX request for searching is made - whether on page load or when I click the button.

Request as copied from Chrome:
```bash
curl 'https://api.bountysource.com/search/bounty_search?access_token=48998.1487395077.01deba342bad0d5727938df679b31e034a3dffdd&callback=CORS&direction=desc&languages=&order=bounty_total&page=1&per_page=50&trackers=' -H 'Pragma: no-cache' -H 'Origin: https://www.bountysource.com' -H 'Accept-Encoding: gzip, deflate, sdch, br' -H 'Accept-Language: en-US,en;q=0.8' -H 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36' -H 'Accept: application/vnd.bountysource+json; version=1' -H 'Referer: https://www.bountysource.com/bounties/search?direction=desc&order=bounty_total&languages=&trackers=&per_page=50&page=1' -H 'Connection: keep-alive' -H 'Cache-Control: no-cache' --compressed
```

Response:
```json
/**/ CORS({"data":{"error":"Internal server error."},"meta":{"status":500,"success":false,"pagination":null}})
```

Note that the response itself actually has a 200 error code.  The 500 seems to be coming from an internal API to the middleware.

Full response headers:
```http
HTTP/1.1 200 OK
Access-Control-Allow-Headers: Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since, X-Requested-With
Access-Control-Allow-Methods: GET, POST, PATCH, PUT, DELETE
Access-Control-Allow-Origin: *
Access-Control-Max-Age: 86400
Cache-Control: no-cache, no-store, max-age=0, must-revalidate
Content-Encoding: gzip
Content-Type: application/javascript; charset=utf-8
Expires: Fri, 01 Jan 1990 00:00:00 GMT
Pragma: no-cache
Vary: Accept-Encoding
X-Content-Type-Options: nosniff
X-Frame-Options: SAMEORIGIN
X-Request-Id: 0365d720-0242-412f-ac8c-340b821adfd8
X-Runtime: 0.062265
X-UA-Compatible: chrome=1
X-XSS-Protection: 1; mode=block
Content-Length: 126
Connection: keep-alive
```

**Comment by frnco** [comment]
An answer from core team would be nice. Any answer...

**Comment by magwas** [comment]
The search page does not work. When I push the search button, the ajax response comes back with 200, but with "Internal server error" as the object. I did not set anything in the search fields.
There is no error message visible for the average Joe user.
pasting here the result of "copy all as HAR":

{
  "log": {
    "version": "1.1",
    "creator": {
      "name": "Firefox",
      "version": "51.0.1"
    },
    "browser": {
      "name": "Firefox",
      "version": "51.0.1"
    },
    "pages": [
      {
        "startedDateTime": "2017-03-26T15:34:10.897+02:00",
        "id": "page_3",
        "title": "Bountysource",
        "pageTimings": {
          "onContentLoad": -1,
          "onLoad": -1
        }
      }
    ],
    "entries": [
      {
        "pageref": "page_3",
        "startedDateTime": "2017-03-26T15:34:10.897+02:00",
        "time": 822,
        "request": {
          "bodySize": 0,
          "method": "GET",
          "url": "https://api.bountysource.com/search/bounty_search?access_token=42700.1490532116.3bf4caeaccfe463b6c1695444decfff361032f16&callback=CORS&direction=desc&languages=&order=bounty_total&page=1&per_page=50&trackers=",
          "httpVersion": "HTTP/1.1",
          "headers": [
            {
              "name": "Host",
              "value": "api.bountysource.com"
            },
            {
              "name": "User-Agent",
              "value": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:51.0) Gecko/20100101 Firefox/51.0"
            },
            {
              "name": "Accept",
              "value": "application/vnd.bountysource+json; version=1"
            },
            {
              "name": "Accept-Language",
              "value": "hu,en;q=0.7,es;q=0.3"
            },
            {
              "name": "Accept-Encoding",
              "value": "gzip, deflate, br"
            },
            {
              "name": "Referer",
              "value": "https://www.bountysource.com/bounties/search?direction=desc&order=bounty_total&languages=&trackers=&per_page=50&page=1"
            },
            {
              "name": "Origin",
              "value": "https://www.bountysource.com"
            },
            {
              "name": "Connection",
              "value": "keep-alive"
            }
          ],
          "cookies": [],
          "queryString": [
            {
              "name": "access_token",
              "value": "42700.1490532116.3bf4caeaccfe463b6c1695444decfff361032f16"
            },
            {
              "name": "callback",
              "value": "CORS"
            },
            {
              "name": "direction",
              "value": "desc"
            },
            {
              "name": "languages",
              "value": ""
            },
            {
              "name": "order",
              "value": "bounty_total"
            },
            {
              "name": "page",
              "value": "1"
            },
            {
              "name": "per_page",
              "value": "50"
            },
            {
              "name": "trackers",
              "value": ""
            }
          ],
          "postData": {
            "mimeType": "",
            "params": [],
            "text": ""
          },
          "headersSize": 635
        },
        "response": {
          "status": 200,
          "statusText": "OK",
          "httpVersion": "HTTP/1.1",
          "headers": [
            {
              "name": "Access-Control-Allow-Headers",
              "value": "Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since, X-Requested-With"
            },
            {
              "name": "Access-Control-Allow-Methods",
              "value": "GET, POST, PATCH, PUT, DELETE"
            },
            {
              "name": "Access-Control-Allow-Origin",
              "value": "*"
            },
            {
              "name": "Access-Control-Max-Age",
              "value": "86400"
            },
            {
              "name": "Cache-Control",
              "value": "no-cache, no-store, max-age=0, must-revalidate"
            },
            {
              "name": "Connection",
              "value": "keep-alive"
            },
            {
              "name": "Content-Encoding",
              "value": "gzip"
            },
            {
              "name": "Content-Length",
              "value": "126"
            },
            {
              "name": "Content-Type",
              "value": "application/javascript; charset=utf-8"
            },
            {
              "name": "Expires",
              "value": "Fri, 01 Jan 1990 00:00:00 GMT"
            },
            {
              "name": "Pragma",
              "value": "no-cache"
            },
            {
              "name": "Vary",
              "value": "Accept-Encoding"
            },
            {
              "name": "X-Content-Type-Options",
              "value": "nosniff"
            },
            {
              "name": "X-Frame-Options",
              "value": "SAMEORIGIN"
            },
            {
              "name": "X-Request-Id",
              "value": "6539b68e-4e43-499a-bb27-9e60aaa9573e"
            },
            {
              "name": "X-Runtime",
              "value": "0.045261"
            },
            {
              "name": "X-XSS-Protection",
              "value": "1; mode=block"
            },
            {
              "name": "x-ua-compatible",
              "value": "chrome=1"
            }
          ],
          "cookies": [],
          "content": {
            "mimeType": "application/javascript; charset=utf-8",
            "size": 110,
            "text": "/**/ CORS({\"data\":{\"error\":\"Internal server error.\"},\"meta\":{\"status\":500,\"success\":false,\"pagination\":null}})"
          },
          "redirectURL": "",
          "headersSize": 746,
          "bodySize": 126
        },
        "cache": {},
        "timings": {
          "blocked": 638,
          "dns": 0,
          "connect": 0,
          "send": 0,
          "wait": 184,
          "receive": 0
        },
        "serverIPAddress": "54.81.249.144",
        "connection": "443"
      }
    ]
  }
}

**Comment by frnco** [comment]
@rappo Can be closed, together with #1012
